service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/LocalAuthorityChangedHandler.java

```diff
@@ -74,6 +104,25 @@ public void notifySecondaryLocalAuthority(final SecondaryLocalAuthorityAdded eve
             caseData.getId());
     }
 
+    @Async
+    @EventListener
+    public void notifyDesignatedLocalAuthority(final CaseTransferredToAnotherCourt event) {
+        final CaseData caseData = event.getCaseData();
+
+        final RecipientsRequest recipientsRequest = RecipientsRequest.builder()
+            .caseData(caseData)
+            .secondaryLocalAuthorityExcluded(true)
+            .legalRepresentativesExcluded(true)
+            .build();
+
+        final Set<String> recipients = localAuthorityRecipients.getRecipients(recipientsRequest);
+        if (!recipients.isEmpty()) {
```

Comment:
why don't we don't have this recipients empty check in the above notifySecondaryLocalAuthority method